example/index.tsx

```diff
@@ -24,6 +25,12 @@ const App = () => (
         componentRestrictions: { country: 'gb' },
       },
     }}
+    onMapInit={({ map, infoWindow, autocomplete, originMarker }: StoreLocatorMap) => {
+      console.log(map);
+      console.log(infoWindow);
+      console.log(autocomplete);
+      console.log(originMarker);
+    }}
```

Comment:
Should these `console.log` statements be here?

Reply:
This is the example site that you run locally for developing, so I left it in there intentionally for testing. I should probably add the new `storeList` object though.

Reply:
@dmccabe this PR is still waiting your approval - I can remove the console.logs if you prefer 😂

Reply:
Hah nope were good, was just waiting for a response before approval, I missed the notification.